consensus/polybft/bridge_manager.go

```diff
@@ -370,10 +356,9 @@ func (b *bridgeManager) initCheckpointManager(
 	return nil
 }
 
-// initStateSyncRelayer initializes state sync relayer
-// if not enabled, then a dummy state sync relayer will be used
+// initStateSyncRelayer initializes bridge event relayer
+// if not enabled, then a dummy bridge event relayer will be used
 func (b *bridgeManager) initStateSyncRelayer(
```

Comment:
Rename method?

Reply:
The relayer is part of the Blade-207 task.